packages/elements-dev-portal/src/handlers/getNodeContent.ts

```diff
@@ -0,0 +1,34 @@
+import { Node } from '../interfaces/node';
+
+export const getNodeContent = async (
```

Comment:
In general we prefer colocating related code, rather than having functional folders. As you can probably see we are moving away from having folders like `utils`, `tests`, etc. 

IMO neither this folder or the function add any value, you could just move it into `useGetNodeContent`. Or actually I would consider inlining it, you don't really gain anything by declaring then destructuring parameters again.

Reply:
One use case for keeping them separate is if someone needs to call the handler outside a hook, such as in the NextJS dev-portal: https://github.com/stoplightio/dev-portal/blob/make-it-work/pages/docs/%5BprojectBranchSlug%5D/%5B%5B...nodeSlug%5D%5D.tsx#L61-L66

Reply:
Oh I see. That's why I said we really need to define the public API before we can reason about the code. Can you add the exports that you meant?

Reply:
Done in https://github.com/stoplightio/elements/pull/1191/commits/a04ea35e0dfaa054470cd9cf1a5e5906ea0774f3